backend/core/scripts/listings-importer.ts

```diff
@@ -2,10 +2,22 @@ import * as client from "../types/src/backend-swagger"
 import axios from "axios"
```

Comment:
Can you check how much our listings-importer.ts deviates from upstream?

Reply:
They differ significantly - I filed https://github.com/CityOfDetroit/bloom/issues/384 with thoughts and to track reconciling them.